robot/Cumulus/tests/browser/donations/delete_opportunity.robot

```diff
@@ -14,21 +14,14 @@ Suite Teardown  Delete Records and Close Browser
 ***Keywords***
 # Sets up all the required data for the test based on the keyword requests
 Setup Test Data
-    &{Data} =  Setup Data
-               ...           contact1=&{contact1_fields}
-               ...           opportunity for contact1=&{opportunity_fields}
-
-    Set suite variable    &{Data}
+    &{data}=  Setupdata   contact   contact_data=&{contact1_fields}     opportunity_data=&{opportunity_fields}
+    Set suite variable    &{data}
```

Comment:
same comment as before: consider calling `set_suite_variable` inside of `setupdata`. You don't have to, but I think it would make the tests just a little cleaner.